pom.xml

```diff
@@ -462,8 +462,15 @@ under the License.
                                         submodules, ${flink.version} will be resolved as the actual Flink version.
                                     -->
                                     <include>org.apache.flink:flink-shaded-force-shading</include>
+                                    <include>org.apache.flink:flink-shaded-guava</include>
                                 </includes>
                             </artifactSet>
+                            <relocations>
+                                <relocation>
+                                    <pattern>flink.shaded.guava</pattern>
```

Comment:
I'm not an expert of `maven-shade-plugin`, but I assume that the pattern should start with `org.apache.flink`. Please correct me if I'm wrong.

Reply:
Done it.(By the way, I used to use `com.google.guava` rather than flink's shaded guava, but there are lots of limit in `checkstyle.xml`):
```xml
<module name="Regexp">
		<property name="format" value="import com\.google\.common\.base\.Preconditions"/>
		<property name="illegalPattern" value="true"/>
		<property name="message"
			value="Use Flink's Preconditions instead of Guava's Preconditions"/>
</module>
```
or

```xml
		<module name="IllegalImport">
			<property name="illegalPkgs" value="com.google.common"/>
			<message key="import.illegal" value="{0}; Use flink-shaded-guava instead."/>
		</module>

```

Reply:
Thanks for the update! You are correct. Depending on Guava directly is prohibited in checkstyle to avoid using non-relocated guava by mistake.